include/odp/api/spec/schedule.h

```diff
@@ -45,6 +45,110 @@ extern "C" {
  * Maximum schedule group name length in chars including null char
  */
 
+/**
+ * Schedule configuration
+ * When schedule configuration is not called by the application, the scheduler
+ * behaves in flow-unaware mode and event flow hash values are ignored
+ */
+typedef struct odp_schedule_config_t {
+	/** Number of flows per queue to be supported
+	 * Scheduler enables flow aware mode when flow count is configured
+	 * greater than 1.
+	 *
+	 * Flows are lightweight entities and packets can be assigned to
+	 * specific flows by the application using odp_event_flow_hash_set()
+	 * before enqueuing the packet into the scheduler. This value is
+	 * ignored unless scheduler supports flow aware mode.
+	 *
+	 * Depening on the implementation this number might be rounded up to
+	 * nearest supported value (e.g power of 2). odp_schedule_config() will
+	 * update passed config with the value actually used. This number
+	 * should be less than maximum flow supported by the implementation.
+	 *
+	 * @see odp_schedule_capability_t
+	 */
+	uint32_t flow_count;
```

Comment:
Again, this argues that we're dealing with flow ids (not hashes) that are in the range 0..`flow_count`-1.

Reply:
1) Most APIs use "num" instead of "count". So, this should be "num_flows"
2) Flow ID is on event level, so all references to "packets" should be converted to "events".
3) Maybe valid range needs to specified.
   * Application can set this to 1 ... capa.max_flows
   * Implementation may round up actual num_flows (how / where to read it)
   * Application/pktin/etc uses flow IDs between 0 and actual "num_flows" - 1